### DISCUSSION TITLE
How do I call the modules registered in the node environment in the electron-main environment？

### DISCUSSION THREAD

**Discussion by zhaomenghuan** [eclipse-theia/theia#8975] [Q&A] [answered]
I think there is browser/electron-browser in the official code that calls the modules registered in the node or electron-main environment, but I don't know how to call the modules registered in the node environment in the electron-main environment. 
Take ApplicationServer as an example. It cannot be called directly in the electron-main environment. Is there any way to call it?

**Comment by kittaakos** [comment, marked as answer]
It's not supported: https://github.com/eclipse-theia/theia/pull/7964#issuecomment-638912058.
